policies/project-directorship/board-meeting-substitutes.md

```diff
@@ -0,0 +1,17 @@
+# Rust Foundation Board Meeting Substitutes
+
+The [Rust Foundation bylaws](https://foundation.rust-lang.org/policies/bylaws) do not give any instructions on how a substitute for a Project Director who cannot make a Foundation Board meeting should be selected. Only the following is state in section 4.3(h):
+
+> Each Director may designate in writing or by electronic transmission to the Chairperson or Secretary (which designation may be withdrawn in writing at any time by such Director or Member) an individual to act as a Director in their stead, whether for a single meeting or as a standing alternate. Any such alternate Director shall be entitled to (i) attend and vote at all meetings which the designating Director does not attend, (ii) sign all written consents in lieu of the designating Director, and (iii) otherwise exercise the duties and enjoy the privileges of the designating Director in the absence or unavailability of the designating Director; provided, however, that no such alternate Director may propose a vote or vote upon any Committee of the Board.
+
+The Rust project, however, sets the following policy for how a substitute for a Project Director is determined:
+
+* The Project Director should inform the other Project Directors and the Council of their absence as soon as possible preferably at least 7 days before the Foundation board meeting will take place.
+* The absence of the Project Director may be announced publicly.
+* Any Project Director or Council member may nominate any Project member who would otherwise be eligible to be a Project Director (with limited exceptions[^demo-exception]).
```

Comment:
It's probably worth being explicitly clear about what eligibility means here? Is the *only* exception demographics?